src/server/service/kv.rs

```diff
@@ -1263,9 +1263,29 @@ fn handle_batch_commands_request<E: Engine>(
                 .map_err(|_| GRPC_MSG_FAIL_COUNTER.kv_pessimistic_rollback.inc());
             response_batch_commands_request(id, resp, tx, timer);
         }
+        Some(BatchCommandsRequest_Request_oneof_cmd::Empty(req)) => {
+            let timer = GRPC_MSG_HISTOGRAM_VEC.invalid.start_coarse_timer();
+            let resp = future_handle_empty(req)
+                .map(oneof!(BatchCommandsResponse_Response_oneof_cmd::Empty))
+                .map_err(|_| GRPC_MSG_FAIL_COUNTER.invalid.inc());
+            response_batch_commands_request(id, resp, tx, timer);
```

Comment:
Could you add a test?

Reply:
There is already a test case in `tests/integrations/server/kv_service.rs`.

Reply:
I mean the `Empty` request.

Reply:
I think it's not necessary. `Empty` itself is simple enough.